libsm will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libsm it will fail in the following way:

```
2024/11/12 23:12:24 INFO running step "test/pkgconf"
2024/11/12 23:12:24 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:24 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:24 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:24 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:24 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + basename /home/build/melange-out/libsm-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + dev_pkg=libsm-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + apk info -L libsm-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + grep -q ^Name: usr/lib/pkgconfig/sm.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + basename usr/lib/pkgconfig/sm.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + lib_name=sm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + echo usr/lib/pkgconfig/sm.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + grep -q '^usr/lib/pkgconfig/sm.pc$\|^usr/share/pkgconfig/sm.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + pkgconf --exists sm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + grep -q ^Version: usr/lib/pkgconfig/sm.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + pkgconf --modversion sm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 INFO 1.2.4 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + grep -q ^Libs: usr/lib/pkgconfig/sm.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + pkgconf --libs sm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 INFO -lSM uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + grep -q ^Cflags: usr/lib/pkgconfig/sm.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN + pkgconf --cflags sm uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN Package ice was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN Perhaps you should add the directory containing `ice.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN Package 'ice', required by 'sm', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 WARN Package 'xproto', required by 'sm', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:24 INFO pod bd3d379e4ad120291cff7116ec0795b20cfa180ae49b34ba8aebeea6197cdb01 terminated
2024/11/12 23:12:24 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:24 INFO   workspace dir: /home/user/tmp/melange-workspace-2969659504
2024/11/12 23:12:24 INFO   guest dir: /home/user/tmp/melange-guest-2252217449
2024/11/12 23:12:24 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libsm] Error 1
```

--- a/libsm.yaml
+++ b/libsm.yaml
@@ -56,9 +56,14 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - libice-dev
         - libsm
         - util-linux-dev
+        - xorgproto
     description: libsm dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libsm-doc
     pipeline:

--- a/libsm.yaml
+++ b/libsm.yaml
@@ -1,7 +1,7 @@
 package:
   name: libsm
   version: 1.2.4
-  epoch: 4
+  epoch: 5
   description: X11 Session Management library
   copyright:
     - license: MIT
@@ -56,10 +56,8 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
-        - libice-dev
         - libsm
         - util-linux-dev
-        - xorgproto
     description: libsm dev
     test:
       pipeline:

